next-packages/core/src/style.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+import * as React from 'react'
+import {
+  getRegisteredStyles,
+  insertStyles,
+  isBrowser,
+  getClassName
+} from '@emotion/utils'
+import { serializeStyles } from '@emotion/serialize'
+import { withCSSContext } from '@emotion/core'
+
+type ClassNameArg =
+  | string
+  | boolean
+  | (() => ClassNameArg)
+  | { [key: string]: boolean }
+  | Array<ClassNameArg>
+
+let classnames = (args: Array<ClassNameArg>): string => {
```

Comment:
can u explain more why this is duplicated? I suspect it's like a global caching registry issue, but couldnt this share the implementation with original cx, css & merge?

Reply:
I’m just lazy, I think I’ll move it and merge into `@emotion/utils`. There’s no point in sharing css and cx since they need slightly different things in here and in create-emotion.